src/vs/workbench/contrib/files/browser/files.contribution.ts

```diff
@@ -345,13 +345,13 @@ configurationRegistry.registerConfiguration({
 	'properties': {
 		'explorer.openEditors.visible': {
 			'type': 'number',
-			'description': nls.localize({ key: 'openEditorsVisible', comment: ['Open is an adjective'] }, "The maximum number of editors shown in the Open Editors pane. Setting this to 0 hides the Open Editors pane."),
+			'description': nls.localize({ key: 'openEditorsVisible', comment: ['Open is an adjective'] }, "The initial maximum number of editors shown in the Open Editors pane. Exceeding this amount will show a scroll bar, and allow resizing the pane past this value if the user prefers. Setting this to 0 hides the Open Editors pane."),
```

Comment:
How about "Exceeding this limit will show a scroll bar and allow resizing the pane to display more items."